src/Components/Home.js

```diff
@@ -1,8 +1,9 @@
 import './Home.css'
 
-import React, { useState, useEffect } from 'react'
-import { ProgressBar } from 'primereact/progressbar'
+import React, { useEffect, useState } from 'react'
+
```

Comment:
I  recommend removing this line.
```suggestion
```

Reply:
@Panquesito7 the `PrgressBar` component is used in line 26.

Reply:
> @Panquesito7 the `PrgressBar` component is used in line 26.

My suggestion is to remove line 4, which is an empty line. 🙂

Reply:
Sorry, I miss understood your suggestion :)

Reply:
Please mark these things at resolved.. after they are resolved